### ISSUE TITLE
Add relationship in email to validators

### ISSUE THREAD

**Issue by rethore** [FractalFlows/Competipedia#10]
In the email that the validators receive there is no mention about to which company the proposed company should be the competitor to. This should be visible in the email so that the validator accept/reject the company accordingly.

**Comment by IAbda** [comment]
agreed. The original list of competitors should be listed in the email sent.

**Comment by guilhermedecampo** [comment]
ok
